x-pack/plugins/lens/common/expressions/heatmap_chart/types.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HeatmapGridConfigResult } from './heatmap_grid';
+import { HeatmapLegendConfigResult } from './heatmap_legend';
+
+export type ChartShapes = 'heatmap';
```

Comment:
This type has been simplified from `typeof CHART_SHAPES[keyof typeof CHART_SHAPES]` to just `heatmap`. Seems fine but it's the only change to heatmap types I can find

Reply:
Correct. I assumed that in case of generalisation it could be augmented later on here.